contributing.md

```diff
@@ -4,16 +4,42 @@ Please note that this project is released with a [Contributor Code of Conduct](c
 
 ---
 
-Ensure your pull request adheres to the following guidelines:
-
-<!--
-- Make sure you take care of this
-- And this as well
-- And don't forget to check this
--->
+## App to be submitted
 
 Thank you for your suggestions!
 
+Note, however, that not everything will be considered. The aim of the list is to provide a concise set of awesome Bash scripts and resources. This means that suggestions should, where applicable:
+
+- Use or refer to Bash directly (i.e. there are more appropriate Awesome Lists tracking general CLI tools).
+- Do one thing and do it well.
+- Have a free and open source license.
+- Be easy to install.
+- Be well documented.
+- Have a decently active community and engaged ownership.
+
+## Pull Request Style
+
+#### Ground Rules
+
+- No duplicates.
+- Individual pull request for each suggestion.
+- The repository is older than 90 days (where applicable).
+- The repository has more than 10 stars (where applicable).
```

Comment:
Could we request something like 50 stars? 30? less? more? The main goal of awesome list is to show what the community find awesome. I think it is quite fair to request some "approvals".

For non repository, we could ask for reddit/HN/lobsters link. 

WDYT?

Reply:
cc @Knusper

Reply:
Yep, I think it'd be fine to start out higher and drop it if we find it too restrictive. I'll bump it to 50.

That's an excellent idea re:reddit/hn/lobsters link for corroboration against non-source items. I'll definitely add that, too.